doc/CONFIGURATION.md

```diff
@@ -227,6 +228,32 @@ ExecStop=/usr/bin/fusermount -u /home/ec2-user/s3-bucket-mount
 WantedBy=remote-fs.target
 ```
 
+## Caching configuration
+
+Mountpoint can optionally cache object metadata and content to reduce cost and improve performance for repeated reads to the same file.
+
+To enable caching, use the `--cache <CACHE_DIR>` command-line flag.
+This flag will enable caching of metadata using a default time-to-live (TTL) of 1 second.
+Object content will also be cached within the cache directory specified.
+Mountpoint will create a directory within this path, removing any files or directories within it at mount time and exit.
+When running multiple Mountpoint processes on the same host,
+you should use unique cache directories to avoid different processes interfering with the others' cache content.
+
+Mountpoint caching can be further configured,
+such as adjusting the metadata time-to-live (TTL) or the maximum space allowed to be used by the data cache.
+Review the caching options available using `mount-s3 --help`.
```

Comment:
It may be worth calling out the metadata TTL specifically here also, since we expect this to be something customers may want to tune.